src/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const Button = (props) => {
```

Comment:
Good, Kadir.
Note that with arrow functions where you're simply returning a single containing JSX element, you can use implicit return and also omit the braces, which makes things a bit more readable.  E.g.: 

```
const MyComponent = () =>     
  <div>
    <Greeting />
    <Mentor />
  </div>
```